PSReadLine/PSReadLine.format.ps1xml

```diff
@@ -147,6 +147,10 @@ $d = [Microsoft.PowerShell.KeyHandler]::GetGroupingDescription($_.Group)
               <ListItem>
                 <PropertyName>ViModeIndicator</PropertyName>
               </ListItem>
+              <ListItem>
```

Comment:
Since the value will almost always be `$null`, I think it should be excluded by default, which can be done with `ItemSelectionCondition`.

And I think the script block text will be truncated, so maybe this is more reasonable?

```
              <ListItem>
                <Label>ViModeChangeHandler</Label>
                <ItemSelectionCondition><ScriptBlock>$null -ne $_.ViModeChangeHandler</ScriptBlock></ItemSelectionCondition>
                <PropertyName>ViModeChangeHandler</ScriptBlock>
              </ListItem>
```